src/app/ui/icons/logomark-icon.tsx

```diff
@@ -0,0 +1,12 @@
+import { Icon, IconProps } from './icon';
+
+export function LogomarkIcon({ color = 'currentColor', strokeWidth = '2', ...props }: IconProps) {
```

Comment:
Why `LogomarkIcon` instead of `LogoIcon`?

Homage to Mark?

Reply:
This was determined by the design team. We have a `lettermark` and a `logomark`. To distinguish b/w the `L` and `Leather`.

Reply:
OK cool, thanks!